### PR TITLE
Create the AI/ML german takeover

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Build the AI/ML webinar takeover in German
Fixed the existing takeovers to work in the new system.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Set you browser lang to German
- Refresh the homepage until you see the German AI takeover
- Check it matches [the copy doc](https://docs.google.com/document/d/1CQNGRpldiJ1js5fRczKc8kDDZOlrVyNkfYIJ3r8OCVc/edit)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4809

### CODE REVIEW

File: templates/takeovers/_german_ai_webinar.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<section lang="de" class="p-strip is-deep p-takeover--ai-webinar js-takeover">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1 class="p-takeover__title">Alles, was Sie über Ubuntu, Sicherheit und Compliance wissen müssen</h1>
+      <p class="p-takeover__text">Erfahren Sie mehr über Ubuntu, eine führende Technologie für regulatorische Anforderungen in verschiedensten Branchen.</p>
+      <div class="u-hide--small">
+        <a href="https://www.brighttalk.com/webcast/17086/351767?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_DC_DACH_OpenStack_WBN_ComplianceDE_takeover" class="p-button--positive u-no-margin--bottom"><span class="p-link--external">Webinar ansehen</span></a>
```
Comment (pmahnke) [inline templates/takeovers/_german_ai_webinar.html]:
```suggestion
        <a href="https://www.brighttalk.com/webcast/17086/352220?utm_source=takeover&utm_medium=takeover&utm_campaign=CY19_DC_DACH_Kubernetes_WBN_AIMLDE_Takeover " class="p-button--positive u-no-margin--bottom"><span class="p-link--external">Webinar ansehen</span></a>
```

The copy doc is different

**Comment by anthonydillon** [comment]
@pmahnke sorry updated the copy doc and linked issue.

**Comment by pmahnke** [comment]
@anthonydillon I think the text is still wrong and my suggestions correct

**Comment by anthonydillon** [comment]
@pmahnke thanks updated.

**Pull request merged**